docsy-example/README.md

```diff
@@ -0,0 +1,101 @@
+# Docsy Example
+
+[Docsy](https://github.com/google/docsy) is a Hugo theme for technical documentation sites, providing easy site navigation, structure, and more. This **Docsy Example Project** uses the Docsy theme, as well as providing a skeleton documentation structure for you to use. You can either copy this project and edit it with your own content, or use the theme in your projects like any other [Hugo theme](https://gohugo.io/themes/installing-and-using-themes/).
+
+This Docsy Example Project is hosted at [https://example.docsy.dev/](https://example.docsy.dev/).
+
+You can find detailed theme instructions in the Docsy user guide: https://docsy.dev/docs/
+
+This is not an officially supported Google product. This project is currently maintained.
+
+## Cloning the Docsy Example Project
+
+The following will give you a project that is set up and ready to use (don't forget to use `--recurse-submodules` or you won't pull down some of the code you need to generate a working site). The `hugo server` command builds and serves the site. If you just want to build the site, run `hugo` instead.
+
+```bash
+git clone --recurse-submodules --depth 1 https://github.com/google/docsy-example.git
+cd docsy-example
+hugo server
+```
+
+The theme is included as a Git submodule:
+
+```bash
+▶ git submodule
+ a053131a4ebf6a59e4e8834a42368e248d98c01d themes/docsy (heads/master)
+```
+
+If you want to do SCSS edits and want to publish these, you need to install `PostCSS` (not needed for `hugo server`):
+
+```bash
+npm install
+```
+
+<!--### Cloning the Example from the Theme Project
+
+
+```bash
+git clone --recurse-submodules --depth 1 https://github.com/docsy.git
+cd tech-doc-hugo-theme/exampleSite
+HUGO_THEMESDIR="../.." hugo server
+```
+
+
+Note that the Hugo Theme Site requires the `exampleSite` to live in a subfolder of the theme itself. To avoid recursive duplication, the example site is added as a Git subtree:
+
+```bash
+git subtree add --prefix exampleSite https://github.com/google/docsy.git  master --squash
+```
+
+To pull in changes, see `pull-deps.sh` script in the theme.-->
+
+## Running the website locally
+
+Once you've cloned the site repo, from the repo root folder, run:
+
+```
+hugo server
+```
+
+### Running a container locally
+
+You can run docsy-example inside a [Docker](ihttps://docs.docker.com/)
```

Comment:
nit:
```suggestion
You can run docsy-example inside a [Docker](https://docs.docker.com/)
```

Reply:
I can check if I created this typo or if the upstream `README.md` has a typo (log an issue with the theme)?

Reply:
Ah it might be that!